Charter.md

```diff
@@ -0,0 +1,135 @@
+# PlanetarPy Charter
+
+PlanetaryPy is governed by the members of the Technical Committee (TC).
+
+## Statement of Purpose
+
+PlanetaryPy is a community effort to develop a core package
+for planetary science in Python and foster interoperability between
+Python planetary science packages.
+
+## Responsibilities
+
+The PlanetaryPy TC responsibilities include:
+
+* Technical direction
+* Setting release dates.
+* Release quality standards.
+* Project governance and process (including this policy)
+* Contribution policy
+* GitHub repository hosting
+* Conduct guidelines
+* Maintaining the list of additional Collaborators
```

Comment:
lower case collaborators?

Reply:
or not as I see it this way elsewhere..

Reply:
Yeah, using 'capital C' Collaborators signifies that we mean something specific in this document rather than or in addition to the dictionary definition of collaborator.